src/main/scala/swiss/dasch/infrastructure/Metrics.scala

```diff
@@ -16,6 +16,6 @@ object Metrics {
       ZLayer.succeed(MetricsConfig(interval = 5.seconds)),
       prometheus.publisherLayer,
       prometheus.prometheusLayer,
-      DefaultJvmMetrics.live.unit.orDie,
+      DefaultJvmMetrics.liveV2.unit.orDie,
```

Comment:
This will change the prometheus format and may break integration with grafana and other reporting, did you clarify this?

Reply:
Nope, I will take a look at this. Since it's a deprecation warning, I immediately assumed it wouldn't be a breaking change.

Reply:
There doesn't seems to be no way to keep using the old functionality, as far as I can tell.